src/statusBar.ts

```diff
@@ -43,14 +48,31 @@ export class StatusBar {
   }
 
   private async updateScarbVersion(): Promise<void> {
-    try {
-      const scarb = await Scarb.find(vscode.workspace.workspaceFolders?.[0], this.context);
-      if (scarb) {
-        const version = await scarb.getVersion(this.context);
-        this.statusBarItem.tooltip = `Cairo Language\n${version}`;
+    if (this.client) {
+      try {
+        const request = this.client.sendRequest(lsVersions);
+
+        // TODO(#50) When there is no handler for method on server it never resolves instead of failing.
+        const response = await timeout(request, 1000);
+
+        this.statusBarItem.tooltip = `Cairo Language Server ${response.ls.version} (${response.ls.path})`;
+
+        if (response.scarb) {
+          this.statusBarItem.tooltip += `\n${response.scarb.version}`;
+        }
+      } catch {
+        this.context.log.error(`catched fail response`);
```

Comment:
```suggestion
        this.context.log.trace("Status bar: Falling back to old version resolution");
```
(if we want to keep this)

Reply:
We don't want but we have to, older LS does not support this new custom method we use here.